packages/webpack/mixins/stats/mixin.core.js

```diff
@@ -29,14 +29,9 @@ class WebpackStatsMixin extends Mixin {
       const { StatsPlugin } = require('../../lib/plugins/stats');
       plugins.unshift(new StatsPlugin(this.statsPromise));
     }
-    const isStaticBuild =
-      this.options.static &&
-      (process.env.NODE_ENV === 'production' ||
-        this.options._.includes('build'));
-    if (
-      target === 'node' &&
-      (isStaticBuild || process.env.NODE_ENV === 'production')
-    ) {
+    const isBuild =
+      this.options._.includes('build') || process.env.NODE_ENV === 'production';
+    if (target === 'node' && isBuild) {
```

Comment:
This is supposed to catch all conditions that are the opposite of: `mode === 'develop'` as is used here:
- https://github.com/untool/untool/blob/master/packages/webpack/mixins/render/mixin.core.js#L38
- https://github.com/untool/untool/blob/master/packages/webpack/mixins/render/mixin.core.js#L41